support listArchives with sessionId, fixes #122

kayhuba opened the pull request:
As @davideberlein already mentioned we're in need of this feature. Please review the PR and advise, thanks.

Regards
Kay

aiham commented on src/main/java/com/opentok/util/HttpClient.java:
```diff
@@ -133,6 +133,28 @@ public String getArchives(int offset, int count) throws RequestException {
+        try {
+            Response response = request.get();
+            switch (response.getStatusCode()) {
+            case 200:
```
Could you indent everything in switch statement once?

aiham replied:
Could you indent everything in switch statement once?

kayhuba commented:
@aiham Thanks for the review! I've changed the base of the PR to dev now and fixed the indentation problem with the switch statement. Can you have another look?

aiham commented:
Version 3.1.0 has been published. Thanks @kayhuba

kayhuba commented:
Many thanks @aiham !
